app.js

```diff
@@ -198,7 +199,9 @@ async function getGeostatsData() {
 
         return GeoConsumerContract.getGeostatsData()
             .then(geostats_data => {
-                return geostats_data.toString()   // Converting to String representation of BigNumber _hex (already in the 18-decimal integer representation)
+
+                const buffer = ethers.utils.parseUnits("1", "ether")
```

Comment:
Read the "1" that you have currently hard-coded by reading the `shift` parameter from the JSON file (to be added)